frontend/packages/helm-plugin/src/components/forms/install-upgrade/HelmInstallUpgradePage.tsx

```diff
@@ -41,40 +40,45 @@ const HelmInstallUpgradePage: React.FunctionComponent = () => {
   const searchParams = new URLSearchParams(location.search);
   const [activePerspective] = useActivePerspective();
 
-  const namespace = params.ns || searchParams.get('preselected-ns');
+  const namespace = params.ns ?? searchParams.get('preselected-ns');
   const initialChartURL = searchParams.get('chartURL');
   const indexEntry = searchParams.get('indexEntry');
-  const initialReleaseName = params.releaseName || '';
+  const initialReleaseName = params.releaseName ?? '';
   const helmChartName = searchParams.get('chartName');
   const helmChartRepoName = searchParams.get('chartRepoName');
-  const helmActionOrigin = searchParams.get('actionOrigin') as HelmActionOrigins;
+  const helmActionOriginParam = searchParams.get('actionOrigin');
+  const helmActionOrigin =
+    helmActionOriginParam &&
+    Object.values(HelmActionOrigins).includes(helmActionOriginParam as HelmActionOrigins)
+      ? (helmActionOriginParam as HelmActionOrigins)
+      : undefined;
 
   const { t } = useTranslation();
-  const [chartData, setChartData] = React.useState<HelmChart>(null);
+  const [chartData, setChartData] = React.useState<HelmChart | null>(null);
   const [chartName, setChartName] = React.useState<string>('');
   const [chartVersion, setChartVersion] = React.useState<string>('');
   const [appVersion, setAppVersion] = React.useState<string>('');
   const [chartReadme, setChartReadme] = React.useState<string>('');
   const [chartHasValues, setChartHasValues] = React.useState<boolean>(false);
-  const [chartError, setChartError] = React.useState<Error>(null);
+  const [chartError, setChartError] = React.useState<Error | null>(null);
 
   const [initialYamlData, setInitialYamlData] = React.useState<string>('');
-  const [initialFormData, setInitialFormData] = React.useState<object>();
+  const [initialFormData, setInitialFormData] = React.useState<object | undefined>();
   const [initialFormSchema, setInitialFormSchema] = React.useState<JSONSchema7>();
   const helmAction: HelmActionType = initialChartURL
     ? HelmActionType.Create
     : HelmActionType.Upgrade;
 
-  const config = React.useMemo<HelmActionConfigType>(
+  const config = React.useMemo(
     () =>
       getHelmActionConfig(
         helmAction,
         initialReleaseName,
-        namespace,
+        namespace ?? '',
```

Comment:
What happens when you call this method with no namespace? (Or `''` for the other values below?)